benchmark/README.md

```diff
@@ -34,6 +34,10 @@ benchmarker to be used by providing it as an argument, e. g.:
 
 `node benchmark/http/simple.js benchmarker=autocannon`
 
+Basic Unix tools are required for some benchmarks.
+[Git for Windows][git-for-windows] includes Git Bash and tools which can be
+included in the global `PATH`.
```

Comment:
Maybe `Git for Windows includes Git bash and the necessary tools, which need to be included in the global Windows path`